src/main/java/com/smartbear/readyapi/plugin/git/GitCommandHelper.java

```diff
@@ -291,7 +292,7 @@ private boolean isSuccessfulPush(Iterable<PushResult> resultIterable) {
             final RemoteRefUpdate.Status status = refUpdate.getStatus();
             if (status != RemoteRefUpdate.Status.OK && status != RemoteRefUpdate.Status.UP_TO_DATE) {
                 isPushSuccessful = false;
-                logger.warn("Push to one of the remote " + refUpdate.getSrcRef() + " was not successful: " + status);
+                Logging.getErrorLog().error(pushResult.getMessages());
```

Comment:
Shouldn't we concatenate ``pushResult.getMessages()`` with the initial warning ``"Push to one of ..."``?

Reply:
Push result contains this info